README.md

```diff
@@ -0,0 +1,184 @@
+# Awesome AI based Protein Design
+
+
+This is a collection of research papers for **AI-based Protein Design**.
+And the repository will be continuously updated to track the frontier of AI based protein design.
+
+Welcome to follow and star!
+
+
+## Table of Contents
+
+- [A Overview / Example of AI based Protein Design](#a-Overview/Example-of-AI-based-Protein-Design)
+- [Papers](#papers)
+
+  - [Nature](#nature) (**<font color="red">Important!!!</font>**)
+  - [Nature Biomedical Engineering](#nature-biomedical-engineering)
+  - [Nature Communications](#nature-communications)
+  - [Science](#science)
```

Comment:
maybe ** science **  's ranking is higher than ** nature biomedical engineering ** and ** nature communicatons ** ?

Reply:
nature xx is sub-journal of nature

Reply:
i know, if sub-journal of nature  has more influence than science ，then i have no problem